### PR TITLE
Update docker file.

### PR DESCRIPTION
**Pull request by shaopengw**

### CODE REVIEW

File: scripts/docker-hub-publish.sh [hunk 1]
```diff
@@ -7,6 +7,6 @@ if [[ -z "$1" ]] ; then
     exit 1
 fi
 
-docker build . -t laminar/laminar-node:$1 -t laminar/laminar-node:latest --no-cache
-docker push laminar/laminar-node:$1
-docker push laminar/laminar-node:latest
+docker build . -t laminardev/laminar-node:$1 -t laminardev/laminar-node:latest --no-cache
```
Comment (xlc) [inline scripts/docker-hub-publish.sh]: can you revert change on this file and add a new `docker-hub-public-dev` script that is similar to this one but also automatically use `git rev-parse --short HEAD` as the tag

Comment (shaopengw) [reply scripts/docker-hub-publish.sh]: Will add `docker-hub-public-dev`. But we still need to keep the changes in this file which rename `laminar/*` to `laminardev/*` no?

Comment (xlc) [reply scripts/docker-hub-publish.sh]: we have both `laminar` and `lamianrdev` and we will use `lamianr` for public released versions, and `lamianrdev` for internal releases

Comment (shaopengw) [reply scripts/docker-hub-publish.sh]: I see...

**Comment by shaopengw** [comment]
@xlc Yep, will do on Acala.

**Pull request merged**